modules/ROOT/pages/managing-users.adoc

```diff
@@ -24,11 +24,11 @@ Configuring SAML for SSO involves:
 
 == Configure SAML in Identity Management
 
-. In Anypoint Platform, sign into the master organization as a user with the *Organization Administrators*.
+. In Anypoint Platform, sign in to the master organization as a user with the Organization Administrator permission.
```

Comment:
nit: You don't sign in to the master organization. You just sign in.

Reply:
```suggestion
. In Anypoint Platform, sign in as a user with the Organization Administrator permission.
```